lua/nvim-treesitter/parsers.lua

```diff
@@ -402,6 +402,15 @@ list.disassembly = {
   maintainers = { "@ColinKennedy" },
 }
 
+list.djot = {
+  install_info = {
+    url = "https://github.com/treeman/tree-sitter-djot",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+  filtype = "dj",
```

Comment:
Typo. And this is not a standard Vim filetype, so this will do nothing.
```suggestion
```

Reply:
Okay, removed.